src/core/symbology/qgsfillsymbollayer.cpp

```diff
@@ -404,7 +404,19 @@ void QgsSimpleFillSymbolLayer::toSld( QDomDocument &doc, QDomElement &element, c
     // <Fill>
     QDomElement fillElem = doc.createElement( QStringLiteral( "se:Fill" ) );
     symbolizerElem.appendChild( fillElem );
-    QgsSymbolLayerUtils::fillToSld( doc, fillElem, mBrushStyle, mColor );
+    // Apply alpha from symbol
+    bool ok;
+    const double alpha { props.value( QStringLiteral( "alpha" ), 0.0 ).toDouble( &ok ) };
```

Comment:
That mean if we don't have the *alpha* properties defined, then the exported color would be fully transparent.

Wouldn't that be better to use QVariant() as a default value so ok would be false

Reply:
Definitely, I'll make the change.